client/homescreen/activity-panel/panels.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import OrdersPanel from './orders';
+
+export function getAllPanels( { countUnreadOrders } ) {
+	const panels = [
+		{
+			title: __( 'Orders', 'woocommerce-admin' ),
+			count: countUnreadOrders,
+			initialOpen: true,
+			panel: (
+				<OrdersPanel hasActionableOrders={ countUnreadOrders > 0 } />
+			),
+		},
+		// Add another panel row here
+	];
+	return panels;
```

Comment:
Tiny thing but the variable declaration here is not needed. Just return the array directly and we can still add items to it easily.

Reply:
This was changed in the commit 287e3cc